staging/src/kubevirt.io/api/roundtrip/compatibility.go

```diff
@@ -0,0 +1,426 @@
+package roundtrip
```

Comment:
nit: this file and the other new files need a license header. See this for reference:

https://github.com/kubevirt/kubevirt/blob/main/pkg/monitoring/rules/alerts/virt-controller.go#L1-L15

Reply:
Done, updated